src/validation/decorators/is_node_notify_object.ts

```diff
@@ -0,0 +1,86 @@
+/*
+* Copyright 2021 HMS Industrial Networks AB
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http: //www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+import {
+    registerDecorator,
+    ValidationOptions,
+    ValidatorConstraint,
+    ValidatorConstraintInterface,
+    ValidationArguments,
+    isString,
+    maxLength,
+    isNumber,
+    min,
+    max,
+    isBoolean
+} from 'class-validator';
+import { constants } from '../../common/constant';
+import { NodeNotifyObject, NodePermissions } from '../../consumer/rpc/v3_3';
+
+function validateTriggerN(triggerN: string | number | boolean) {
+    return (isNumber(triggerN) || isString(triggerN) || isNumber(triggerN));
+}
+
+function validateTriggerT(triggerT: number) {
+    return (isNumber(triggerT) && !min(triggerT, 0) && !max(triggerT, 4294967295));
```

Comment:
!max(triggerT, 4294967295) will always return false, or am I wrong?

if max returns the highest number, the ! of the number is false if I understand it correctly.

Reply:
As long as the number for triggerT is not higher than 4294967295max(triggerT, 4294967295) shall always return false. By using the ! we invert the result, so we expect !max(triggerT, 4294967295) to always be true